redirects-transition.conf

```diff
@@ -529,6 +542,24 @@ rewrite ^/law/policy/internet09/sunlighthearingcomment.pdf https://www.fec.gov/r
 rewrite ^/law/policy/internet09/webmanageremails.pdf https://www.fec.gov/resources/cms-content/documents/webmanageremails.pdf redirect;
 rewrite ^/law/policy/internet09/westcomments072909.pdf https://www.fec.gov/resources/cms-content/documents/7-29-09websitecomments.pdf redirect;
 
+# law/policy/nationwideiereporting redirects 
+rewrite ^/law/policy/nationwideiereporting/draftnationwideiereporting.pdf https://www.fec.gov/resources/cms-content/documents/draftnationwideiereporting.pdf redirect;
```

Comment:
Check spacing on line 546

Reply:
I went back and stuck the changes into notepad to remove the formatting